### ISSUE TITLE
FAILED akka.http.impl.engine.client.NewConnectionPoolSpec.The host-level client infrastructure should surface connection-level and stream-level errors while receiving response entity

### ISSUE THREAD

**Issue by jrudolph** [akka/akka-http#1748]
https://jenkins.akka.io:8498/job/akka-http-nightly-against-akka-master-on-2.12/388/testReport/junit/akka.http.impl.engine.client/NewConnectionPoolSpec/The_host_level_client_infrastructure_should_surface_connection_level_and_stream_level_errors_while_receiving_response_entity/

```
Error Message

timeout (3 seconds) waiting for 1 messages on ErrorFilter(class java.lang.RuntimeException,None,Left(),false)

Stacktrace

java.lang.AssertionError: timeout (3 seconds) waiting for 1 messages on ErrorFilter(class java.lang.RuntimeException,None,Left(),false)
	at akka.testkit.EventFilter.intercept(TestEventListener.scala:114)
	at akka.http.impl.engine.client.ConnectionPoolSpec$$anon$10.<init>(ConnectionPoolSpec.scala:220)
	at akka.http.impl.engine.client.ConnectionPoolSpec.$anonfun$new$10(ConnectionPoolSpec.scala:180)
```

**Comment by johanandren** [comment]
https://jenkins.akka.io:8498/job/akka-http-nightly-against-akka-master-on-2.12/389/consoleFull

**Comment by jrudolph** [comment]
https://jenkins.akka.io:8498/job/akka-http-nightly-against-akka-master-on-2.12/390/testReport/akka.http.impl.engine.client/NewConnectionPoolSpec/The_host_level_client_infrastructure_should_surface_connection_level_and_stream_level_errors_while_receiving_response_entity/

**Comment by jrudolph** [comment]
https://jenkins.akka.io:8498/job/akka-http-nightly-release-10.0-against-akka-2.4-on-scala-2.12/31/testReport/akka.http.impl.engine.client/NewConnectionPoolSpec/The_host_level_client_infrastructure_should_surface_connection_level_and_stream_level_errors_while_receiving_response_entity/

**Comment by jrudolph** [comment]
Probably fixed by now (after 10.0 backports are merged).
